modules/codes.rb

```diff
@@ -113,7 +113,17 @@ def self.modify_codes(user_id, args, option)
                         '3ds'.to_sym => '<:New3DSXL:287651327763283968> **3DSs**:',
                         :nnid => '<:NintendoNetworkID:287655797104836608> **Nintendo Network IDs**:',
                         :switch => '<:Switch:287652338791874560> **Switches**:',
-                        :game => '🎮 **Games**:'
+                        :game => '🎮 **Games**:',
```

Comment:
Not really sure why you'd add a comma, if I remember Ruby gets angry over that. If not ignore this